app/src/org/commcare/dalvik/activities/EntitySelectActivity.java

```diff
@@ -1191,4 +1218,11 @@ private void stopTimer() {
             }
         }
     }
+
+    public void onLocationChanged() {

```

Comment:
Sorta confusing to give this method the same name as `LocationListener.onLocationChanged`. Makes me think that this method implements that interface.